packages/fast-components-react-base/src/breadcrumb/breadcrumb.props.ts

```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
+import {
+    BreadcrumbClassNameContract,
+    ManagedClasses,
+} from "@microsoft/fast-components-class-name-contracts-base";
+
+export interface BreadcrumbManagedClasses
+    extends ManagedClasses<BreadcrumbClassNameContract> {}
+export interface BreadcrumbUnhandledProps extends React.HTMLAttributes<HTMLElement> {}
+export interface BreadcrumbHandledProps extends BreadcrumbManagedClasses {
+    /**
+     * The breadcrumb children
+     */
+    children?: React.ReactNode;
+
+    /**
+     * The aria-label to provide an accessible name for the breadcrumb
+     */
+    label?: string;
+
+    /**
+     * The breadcrumb seprator
+     */
+    separator?: (className: string) => React.ReactNode;

```

Comment:
This could theoretically be optional if no class is provided to the stylesheet

Reply:
+1 good call